tests/Response/ThrowForStatusTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace WpOrg\Requests\Tests\Response;
+
+use WpOrg\Requests\Exception;
+use WpOrg\Requests\Exception\Http;
+use WpOrg\Requests\Response;
+use WpOrg\Requests\Tests\TestCase;
+
+/**
+ * @covers \WpOrg\Requests\Response::throw_for_status
+ */
+final class ThrowForStatusTest extends TestCase {
+
+	/**
+	 * Verify that a redirection does not lead to an exception when redirection is allowed.
+	 *
+	 * @doesNotPerformAssertions
+	 */
+	public function testRedirectAndAllowed() {
+		$response              = new Response();
+		$response->status_code = 302;
+
+		$response->throw_for_status();
```

Comment:
Testing against a "default value" here seems like an potentially problematic assumption in the tests.

For better reliability, I would suggest:
- always using explicit values for each call
- adding one extra test for testing the current default value

Reply:
Done (fixed up in the original commit).